File: app/src/main/java/com/mygreenpic/greenpic/LoginActivity.java
package com.mygreenpic.greenpic;

import android.annotation.SuppressLint;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

import androidx.appcompat.app.AppCompatActivity;

public class LoginActivity extends AppCompatActivity {


    @Override
    protected void onCreate(Bundle savedInstanceState){
        super.onCreate(savedInstanceState);
        setContentView(R.layout.login);
    }
    @SuppressLint("WrongViewCast")
    public void Login(View v){
        Toast.makeText(getApplicationContext(), "로그인 중입니다.",Toast.LENGTH_LONG).show();
        Intent intent= new Intent(this, MainActivity.class);
        startActivity(intent);
    }

}

File: app/src/main/java/com/mygreenpic/greenpic/LoginActivity.kt
package com.mygreenpic.greenpic

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.text.style.ClickableSpan

class LoginActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.login)
        ClickableSpan("@+id/login_btn")
    }

    private fun ClickableSpan(s: String) {

    }
}